.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
 - repo: https://github.com/jumanjihouse/pre-commit-hooks
   rev: 3.0.0
   hooks:
-  - id: RuboCop
+  - id: rubocop
```

Comment:
Could you revert this change? (So that the changes are related to the addition of Tuxedo OS)

Reply:
thanks for the review, applied both TODOs.